data/mods/Aftershock/maps/mapgen/prepnet_orchard.json

```diff
@@ -63,7 +63,7 @@
         { "item": "supplies_farming", "x": 3, "y": 9, "chance": 99 }
       ],
       "sealed_item": { "p": { "items": { "item": "farming_seeds", "chance": 100 }, "furniture": "f_plant_harvest" } },
-      "place_monsters": [
+      "place_monster": [
         { "monster": "GROUP_PARK_SCENIC", "x": 12, "y": 14, "chance": 2 },
         { "monster": "GROUP_PrepNet", "x": [ 3, 20 ], "y": [ 16, 20 ], "repeat": 4, "friendly": true }
```

Comment:
That was actually right. `place_monsters` places monsters from a monster group, such as `GROUP_PARK_SCENIC`. `place_monster` places specific monster, such as `mon_deer`.

Reply:
`place_monsters` does not support placing friendly monsters.

`place_monstee` supports monster groups too:
https://github.com/CleverRaven/Cataclysm-DDA/blob/73660c1bb65fe970a794fd3bc0c6f7cc0b702a18/src/mapgen.cpp#L1129